modules/gamelist-ui/src/lib/Input/Input.tsx

```diff
@@ -1,19 +1,41 @@
 import * as React from 'react';
+import { Slot } from '@radix-ui/react-slot';
+import { cva, type VariantProps } from 'class-variance-authority';
 import { cn } from '@src/utils';
 
-export type InputProps = React.InputHTMLAttributes<HTMLInputElement>;
+const inputVariants = cva(
+  'border-input bg-background ring-offset-background placeholder:text-muted-foreground focus-visible:ring-ring flex h-10 w-full rounded-md border px-3 py-2 text-sm file:border-0 file:bg-transparent file:text-sm file:font-medium focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50',
+  {
+    variants: {
+      variant: {
+        default: '',
+        loginSignUp:
+          'w-[290px] px-[15px] focus:outline-none focus-visible:ring-0 focus-visible:ring-offset-0 bg-[#f2f4f7]',
```

Comment:
Same here, remove the hard-coded px values. Also can you remove bg-[#f2f4f7] for bg-background? Or is that specific color needed?

Reply:
@Yzma check this out again:
https://github.com/tienviet10/gamelist-fe-v3/pull/16/commits/581dfb820e32bc4ca2843189f0540c6e22565d64